ml/inference/tflite.go

```diff
@@ -123,6 +125,7 @@ func (loader TFLiteModelLoader) Load(modelPath string) (*TFLiteStruct, error) {
 		interpreterOptions: loader.interpreterOptions,
 		Info:               info,
 		modelPath:          modelPath,
+		Mu:                 sync.Mutex{},
```

Comment:
should be unexported

Reply:
This is the inference package and I am locking it in the builtin package, so it needs to be exported. Should I lock it somewhere else?